tools/content-validation/eng/pipelines/python/start.yml

```diff
@@ -0,0 +1,132 @@
+  default: 'all'
+
+- name: branch
+  displayName: |
+      You can choose the review version of the page for testing, and you need to select the corresponding branch. Example: 'nickwalk/yaml-changes'
+  type: string
+  default: 'main'
+
+- name: language
+  displayName: |
+      You can choose the language for testing, only Python, Java supported now. We plan extent to JavaScript, .NET and Go in future. Example: Python
+  type: string
+  default: 'Python'
+  values:
+  - Python
+  - Java
+  - JavaScript
+  - DotNet
+  - Go
+
+stages:
+- stage: ContentValidationTesting
+  jobs:
+  - ${{ each packageName in split(parameters.packages, ',') }}:
+    - ${{ if ne(parameters.packages, 'all') }}:
+      - template: /eng/pipelines/python/package/${{ packageName }}.yml
+        parameters:
+          GitHubPat: $(GITHUB_PAT)
+          GitHubOwner: $(GITHUB_OWNER)
+          GitHubRepo: $(GITHUB_REPO)
+          Branch: ${{ parameters.branch }}
+
+  - ${{ if eq(parameters.packages, 'all') }}:
+    - template: /eng/pipelines/python/package/run-all-packages.yml
+      parameters:
+        AzureDevOpsPat: $(AZURE_DEVOPS_PAT)
+        GitHubPat: $(GITHUB_PAT)
+        GitHubOwner: $(GITHUB_OWNER)
+        GitHubRepo: $(GITHUB_REPO)
+        Branch: ${{ parameters.branch }}
+
+- stage: TotalIssuesSummary
+  condition: always()
+  jobs:
+  - job: Maintain_Total_Issues_List
+    displayName: Maintain_Total_Issues_List
+    pool:
+      vmImage: ubuntu-latest
+    steps:
+    - task: DownloadPipelineArtifact@2
+      displayName: Download The Last Artifact history-issues-summary
+      inputs:
+        buildType: 'specific'
+        project: '$(System.TeamProject)'
+        pipeline: '$(System.DefinitionName)'
+        buildVersionToDownload: 'latestFromBranch'
+        allowPartiallySucceededBuilds: true
+        allowFailedBuilds: true
+        branchName: 'refs/heads/main'
+        artifactName: 'issues-summary'
+        downloadPath: '$(System.DefaultWorkingDirectory)/Artifacts'
+      continueOnError: true
+
+    - task: DownloadPipelineArtifact@2
+      displayName: Download The Current Artifact 
+      inputs:
+        buildType: 'current'
+        downloadPath: '$(System.DefaultWorkingDirectory)/Artifacts'
+
+    - task: DotNetCoreCLI@2
+      displayName: Build issuer helper project
+      inputs:
+        command: 'build'
+        projects: 'IssuerHelper/IssuerHelper.csproj'
+
+    - script: |
+        organization=$(System.CollectionUri)
+        project=$(System.TeamProject)
+        buildId=$(Build.BuildId)
+  
+        pipelineUrl="${organization}${project}/_build/results?buildId=${buildId}"
+  
+        echo "##vso[task.setvariable variable=pipelineUrl;]$pipelineUrl"
+      displayName: Get the current pipeline url
+
+    - task: DotNetCoreCLI@2
+      displayName: Summary Total Issues of all packages
+      env:
+        PackageName: ${{ parameters.packages }}
+        PipelineRunLink: $(pipelineUrl)
+        GitHubPat: $(GITHUB_PAT)
```

Comment:
WEe should define these as pipeline variables in this yml and then for the PAT we will hook up the variable group to the pipeline once we create the pipeline.

Reply:
The current code is used in our test pipeline. After you create the pipeline, we will adjust the code accordingly based on the location of the PAT.